File: src/main/java/de/thatsich/common/util/logging/ILog.java
package de.thatsich.common.util.logging;

/**
 * @author thatsIch
 * @date 05.03.14.
 */
public interface ILog
{
	void info ( String format, Object... data );

	void warn ( String format, Object... data );

	void debug ( String format, Object... data );

	void trace ( Throwable exception );

	void severe ( String format, Object... data );
}

File: src/main/scala/de/thatsich/minecraft/common/logger/ILog.scala
package de.thatsich.minecraft.common.logger

/**
 *
 *
 * @author thatsIch
 * @since 06.04.2014.
 */
trait ILog
{
	def info(format: String, data: AnyRef*)

	def warn(format: String, data: AnyRef*)

	def debug(format: String, data: AnyRef*)

	def trace(exception: Throwable)

	def severe(format: String, data: AnyRef*)
}
